### ISSUE TITLE
if dayOfWeek == 0

### ISSUE THREAD

**Issue by newbie78** [GeoffZhu/vue-event-calendar#52]
if the day of the week is Sunday, firstDay.getDay () gives 0.
as a result, we lost the first week on the calendar.
replace with this code to fix it (in the cal-panel.vue)
```
    dayList () {
        let firstDay = new Date(this.calendar.params.curYear, this.calendar.params.curMonth, 1)
        let dayOfWeek = firstDay.getDay()
        if (dayOfWeek == 0&&this.calendar.options.weekStartOn > 0) {
          dayOfWeek=7
        }

        let startDate = new Date(firstDay)
        startDate.setDate(firstDay.getDate() - dayOfWeek + this.calendar.options.weekStartOn)
```

**Comment by newbie78** [comment]
here russian localization:
```
  ru: {
    dayNames: ["Вс", "Пн", "Вт", "Ср", "Чт", "Пт", "Сб"],
    monthNames: ["Январь", "Февраль", "Март", "Апрель", "Май", "Июнь", "Июль", "Август", "Сентябрь", "Октябрь", "Ноябрь", "Декабрь"],
    format: 'MM/yyyy',
    fullFormat: 'dd/MM/yyyy',
    dayEventsTitle: 'Все события',
    notHaveEvents: 'События отсутствуют'
  },
```
it would be nice to be able to back up through the config dayEventsTitle and notHaveEvents vars

**Comment by GeoffZhu** [comment]
Thanks, 👍, It's a bug.
But , your code only can fix the bug weekStartOn Sunday. I fix it at v1.4.10
